themes/default/gulpfile.js

```diff
@@ -61,15 +64,43 @@ gulp.task('browserSync', ['make-js', 'make-css'], function() {
 
 
 /**
+ * Create JSON objects from SCSS variables
+ */
+gulp.task('json', function() {
+	return gulp.src('build/sass/utilities/_var-breakpoints.scss')
+		.pipe(sassJson())
+		.pipe(gulp.dest('js/src/'))
+});
+
+
+/**
  * Move pure from node_modules into our distribution folder
  * We don't want to edit this file, as there's no real need with such a small framework
  */
-gulp.task('pure', function() {
+gulp.task('pure', ['json'], function() {
+	var path='js/src/var-breakpoints.json';
+	var mediaQueries = {};
+
+	// Read breakpoints and generate mediaQueries object for rework
+	if (fs.existsSync(path)) {
+	    file = fs.readFile(path, 'utf8', function (err,data) {
+			if (err) {
+				return console.log(err);
+			}
+            data = JSON.parse(data);
+
+			for (var k in data.breakpoints) {
+				mediaQueries[k] = "screen and (min-width:" + data.breakpoints[k] + ")";
+			}
+	    });
+	}
```

Comment:
This bit means that the default is a custom grid. In most cases this shouldn't be needed. When it is, it would be good to have some documentation around it, but I don't think making it standard is a good long term choice. 

I'd rather avoid a recipe that does too much if we can help it. For one, it makes upgrading and maintenance harder, but it will also slow down devs who are new to it.